notebook/static/notebook/js/actions.js

```diff
@@ -563,6 +563,25 @@ define(function(require){
                 env.notebook.save_widget.rename_notebook({notebook: env.notebook});
             }
         },
+        'toggle-all-cells-output-visibility':{
```

Comment:
Thinking out loud...

In the new models we have built, this is controlled by the `IOutputAreaViewModel.collapsed` property. The input area also has a collapsed property. The scrolling behavior is set by the `scrolled` property. I think the action names should follow the names of the new model properties, so:

`toggle-all-cells-output-collapsed`
`toggle-all-cells-output-scrolled`

And eventually:

`toggle-all-cells-input-collapsed`
`toggle-all-cells-input-scrolled`

If we want to change the `collapsed` property of our new models to `visible` we could change the actions here too. But I think that collapsed in a better word as there will still be some UI visible to indicate it is collapsed.